dbt/include/impala/macros/utils/any_value.sql

```diff
@@ -0,0 +1,20 @@
+{#
+# Copyright 2022 Cloudera Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#}
+
+{% macro impala__any_value(expression) -%}
+    {#-- return any value (non-deterministic)  --#}
```

Comment:
Can you add comment similar to this L18 for the other macros so that it is clear what the macro is doing.

Reply:
added